Source/Particles/Collision/MCCScattering.H

```diff
@@ -195,6 +196,8 @@ public:
     *
     * @param[in] energy_cost energy cost of ionization
     * @param[in] mass1 mass of the colliding species
+    * @param[in] ion_vel_std standard deviation of ion velocity distribution; for
```

Comment:
Suggested by @roelof-groenewald:
```suggestion
    * @param[in] standard deviation of ion velocity distribution for each velocity direction;
                 for a Maxwellian distribution it should equal sqrt(kB*T/m)
```